velox/docs/functions/spark/datetime.rst

```diff
@@ -257,6 +257,16 @@ These functions support TIMESTAMP and DATE input types.
 
     Alias for ``unix_timestamp(string, format) -> integer``.
 
+.. spark:function:: to_unix_timestamp(date) -> integer
```

Comment:
Please follow the alphabetical order to add this documentation.

Reply:
done